src/solver/conv_asm_implicit_gemm_bwd_v4r1_dynamic.cpp

```diff
@@ -133,6 +135,10 @@ bool ConvAsmImplicitGemmV4R1DynamicBwd::IsApplicable(const ConvolutionContext& c
         return false;
 
     const auto device_name = ctx.GetStream().GetDeviceName();
+#if WORKAROUND_ISSUE_1146
+    if(device_name == "gfx90a")
```

Comment:
In fact due to line 142, if device name is gfx90a, this solver will return false. So the W/A might not be necessary.

Reply:
+1, it really depends on how explicitly you want it to be (and for later reference to get back to this issue). While technically I agree that it is redundant.

Reply:
Yes, this should be reverted as we have other Solvers for devices that have MAI (and that is why this Solver is not enabled for MI100). #1146 was updated recently.